### DISCUSSION TITLE
Adb input text

### DISCUSSION THREAD

**Discussion by ryandaiswara** [Nain57/Smart-AutoClicker#51] [General Feature Requests]
Can you add adb input text? it should be most powerful autoclicker if this apk can do adb input text 🙏

**Comment by Nain57** [comment]
Hi !

I don't get what do you want to achieve, nor the relation between Smart AutoClicker and adb

**Reply by ryandaiswara** [reply]
Auto clicker with auto input text. Sometimes there's a forms/text fields and we can auto input text with adb input text. For example like Auto Login

**Reply by Nain57** [reply]
I understand the utility of writing a text, which can be a good action to add.
But adb input text requires a host PC to work, which is not acceptable for Smart AutoClicker

**Reply by ghost** [reply]
They say adb but they really mean ProcessBuilder which uses adb syntatic sugar.

**Comment by Nain57** [comment]
Closed as Done, app can write text with the set text action
